applications/frontend/src/app/components/overlay/game/scenes/OverlayScene.ts

```diff
@@ -144,6 +155,9 @@ export class OverlayScene extends Phaser.Scene {
 				mawInfoJsonData,
 				(config.socket.auth as SocketAuth).channel
 			)
+
+			this.donationGoalContainer?.updateProgress(mawInfoJsonData.streamers[(config.socket.auth as SocketAuth).channel])
```

Comment:
this is a dangerous game we play here - request_state should not be emitted by other events from the frontend "MAW_INFO_JSON_DATA_UPDATE" to  avoid unecessary loops, althought it might make sense to have a controlpanel state update when new maw data is available